src/components/Button/Button.tsx

```diff
@@ -1,6 +1,7 @@
 import React, {useCallback, useContext} from 'react';
 
-import {Button as CommonButton, Icon, Platform, StoreBadge} from '@gravity-ui/uikit';
+import {StoreBadge} from '@gravity-ui/components';
```

Comment:
`StoreBadge` seems not to have `lang` prop and you should use `href` instead of `url`

Reply:
Oops, just saw the fix here https://github.com/gravity-ui/page-constructor/pull/476